### PR TITLE
oop homework added

### PR DESCRIPTION
**Pull request by shabaiev**

### CODE REVIEW

File: hw_oop/ex1.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class Rectangle(Shape):
+    def __init__(self, w, h):
+        super().__init__(w, h)
+
+    def area(self):
+        return self.width * self.height
+
+        # (w * h) / 2
```
Comment (konflic) [inline hw_oop/ex1.py]: Remove comments

File: hw_oop/ex1.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class Shape:
+
+    def __init__(self, w, h):
```
Comment (konflic) [inline hw_oop/ex1.py:2]: Here you have space before init, in other classes don't. Leave only one version, for codestyle to be solid.

File: hw_oop/ex2.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class Zebra(Animal):
+    animal_type = "Zebra"
+
+
+d = Dolphin(age=20, name="Chuck")
+z = Zebra("Zelda", 30)
+
+print(d.say())
+print(z.say())
```
Comment (konflic) [inline hw_oop/ex2.py]: This red circle means there is no new line at the end of file. Use PyCharm build in formatter to aavoid this mistakes.

File: hw_oop/ex2.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class Animal:
+    animal_type = 'Any animal'
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+    def say(self):
+        return "Hi, I am {} {} and I am {} !".format(self.animal_type, self.name, self.age)
+        # return f"Hi, I am {self.name} and I am {self.age}!"
```
Comment (konflic) [inline hw_oop/ex2.py]: Remove commented code in final version of your tasks

File: hw_oop/ex1.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        self.width = w
+        self.height = h
+
+
+class Triangle(Shape):
+    def __init__(self, w, h):
```
Comment (konflic) [inline hw_oop/ex1.py]: This is the complete copy of init in base class, should be removed if no changes made.

**Review by konflic — CHANGES_REQUESTED** [review]
Hi, some improvements to be made :)

**Review by konflic — APPROVED** [review]

**Review by shabaiev — COMMENTED** [review]
done

**Comment by shabaiev** [comment]
oop homework

**Pull request closed without merging**